doc/changes/changes_2.6.2.md

```diff
@@ -0,0 +1,55 @@
+# Project Keeper 2.6.2, released 2022-08-11
+
+Code name: Empty dependency updates on windows platform
+
+## Summary
+
+Fixed linfeeds in empty dependency updates on windows platform.  PK now suppresses the section "Dependency Updates" in case there are no dependency updates as initially requested in issue #343.
```

Comment:
```suggestion
Fixed linefeeds in empty dependency updates on windows platform. PK now suppresses the section "Dependency Updates" in case there are no dependency updates as initially requested in issue #343.
```